app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- Great job you've done here, kindly make sure the application asks for confirmation of email by enabling `:confirmable` your user model.
`hint:` you should have something like:
    ```ruby
    	devise :database_authenticatable, :registerable,
    				:recoverable, :rememberable, :validatable, :confirmable
    ````

Reply:
Resolved

Reply:
Thanks for drawing my attention